OpeningTimes.js

```diff
@@ -53,19 +53,22 @@ class OpeningTimes {
       }
     }
 
-    // console.log([date.format(), start.format(), end.format()]);
     return date.isBetween(start, end, null, '[]');
   }
 
+  isClosedAllDay(daysOpeningTimes) {
+    return (daysOpeningTimes.times.some( (t) => { return ( t === 'Closed')}));
```

Comment:
I wonder whether it might be worth lower casing the phrase. I know it comes back as `Closed` from the API but if both that and the expectation were lowered it reduce the chances of a change in the API causing us problems. Might be a bit premature though...